fw/rbcx-coprocessor/platformio.ini

```diff
@@ -11,12 +11,12 @@
 [platformio]
 [env]
 lib_deps =
-    Nanopb
+    RoboticsBrno/RB3204-RBCX-coproc-comm
```

Comment:
Viděl jsem, že je to jen draft zatím, ale pro jistotu zmiňuju -  určitě sem chceš dát přesnou verzi, například ve formátu `https://github.com/RoboticsBrno/RB3204-RBCX-coproc-comm/archive/682eb9072d0fa3de1b3a17c0500362687df5ab7e.zip`, jinak se buďto rozbije build (nekompatibilní verze knihovny) nebo platform.io nestáhne novou verzi knihovny. 

(ZIP a ne git používám protože takhle nepotřebuje mít platform.io dostupnout git binárku).

Reply:
jj dobrá poznámka

Reply:
Je tak